yoyodyne/models/modules/attention.py

```diff
@@ -90,6 +90,6 @@ def score(
         # -> B x seq_len x hidden_size
         m = self.M(concat)
         # -> B x seq_len x 1.
-        scores = self.V(torch.tanh((m)))
+        scores = self.V(torch.tanh(m))
```

Comment:
san check, tanh doesn't have any weirdness where everything as to be a tuple? (kinda like how cat needs an iterable?

Reply:
‘(X)’ isn’t a singleton tuple, that’s written ‘(X,)’. Thus these parentheses are no-op.